submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+Moving on from producing steam to turning it into power, for every heat exchanger, you need 1.718 steam turbines and it is best to round up the product. So in our case, we have 132 * 1.718 = 226.776, or 227 steam turbines. 
+
+And so we have all the component counts covered. If you want to learn more about the mathematics involved here, or if you want a calculator to do most of it for you, the  [Factorio Cheat Sheet](https://factoriocheatsheet.com/#nuclear-power) is the resource I'd recommend.
+
+### What do we consider when placing the components?
+
+When placing down the reactors, we already know about harnessing the neighbor bonus and  calculating it using SRE diagrams. Given that we want to connect as many neighbors as possible, it would make sense to just have a huge spaceless grid of reactors, but the disadvantage of that is there being no space to set up a system for automatically refueling any of the reactors in the middle, unless you use [mods](https://mods.factorio.com/mod/logistics-reactor-next). Hence the most popular reactor arrangement is 2xN, where all reactors except for the corner ones have 3 neighbors, hence each has at least 1 open side for inserter access. However, you can get as creative as you want and leave 2xN behind once you have a steady supply of U-235 such that the neighbor bonus is not so critical anymore.
+
+Your other big concern is fluid flow, and the main constraint is the heat flow along heat pipes. The farther away the heat gets from the reactors, the lower the maximum temperature the pipes can have. Since reactors temperatures cannot exceed 1000C, heat pipes beyond a certain distance become too cold to operate heat exchangers. As the [wiki shows](https://wiki.factorio.com/Tutorial:Nuclear_power#Heat_pipes) and [this post effectively visualizes](https://www.reddit.com/r/factorio/comments/ge4y6c/heat_pipe_throughput_and_a_bonus_note_on_parallel/),  a single pipe can heat a row of at most 21 heat exchangers, but you can improve the range here by making the pipe thicker and by adding heat exchangers on both sides of it instead of one side.  Another tricked used by the community here is placing unfueled nuclear reactors to act as 5x5 heat pipes because this allows travelling the same distance with fewer entities and connecting to more heat exchangers from a single entity. In any case, your best bet is to keep your heat pipes short by keeping heat exchangers nearby. 
+
+As for the other fluids, you want to ensure that the water from your offshore pumps is reaching your heat exhangers without significant [throughput loss in pipelines](https://wiki.factorio.com/Fluid_system#Pipelines), hence make sure to have multiple pipelines along the way and regularly place pumps on them. Secondly, you want to make sure the steam generated has enough pipe space to flow from exchangers to turbines without backing up, which is usually not a big issue and can be solved again using more pipes and pumps.

```

Comment:
what about the obvious solotion of placing your nuclear plant into a lake with a lot of landfill and offshore pumps directly connected to the exchangers? :)
Also, be very careful about advising to place pumps along a pipe line, it have been demonstrated, but never really documented to my knowledge, that they are not a solution to everything, especially if you are doing the ratios properly, even on a long distance. As a reference, you might want to talk to SteelAxe team about their speedruns and how they manage oil setup over long distances. Antielitz did demonstrate placing 200 refineries (if I recall correctly) on a single pipe lane without any intermediate pumps.

Reply:
I think it's enough to mention to simply keep the pipelines short, and use undergrounds where possible. But it's always easiest to build your nuclear reactors on top of, or next to, a lake.

You mentioned the nice table on the wiki too. You could maybe mention, that if the player is going with the example above, and one pipeline per offshore pump, then the player need to ensure the pipeline is not longer than 17 pipe _sections_. Because then the throughput drops below 1200/s, i.e. the offshore pumps capability.

Reply:
Definitely should mention just being close to the water as the primary advice. It also makes sense to specifically mention 17 pipe sections because of 1200/s.

I probably should also warn that pump reliance may have cascading failure if your pumps stop from no power and water runs out from no pumping...

Reply:
maybe one day, someone will do a pump article ^^
my pump's point was they are not working as we expect them to work, i.e. the common mistake is that a factorio pump does not pressurize a pipe, it just empties the side it pulls from. this means that using a pump does not increase the overall throughput of your pipeline, hence why I think any pump advice are sensitive as most people does not understand/know the details going with it :)